docs/README.md

```diff
@@ -1,16 +1,16 @@
-# Contributing to the Service Mesh Performance Specification Documentation
+# Contributing to the Service Mesh Performance Documentation
```

Comment:
This repo has "specification" too

Reply:
Oh, good call. This one needs to be removed, too.